test/click.spec.js

```diff
@@ -405,4 +405,41 @@ module.exports.describe = function({testRunner, expect, playwright, FFOX, CHROMI
       expect(await page.evaluate('document.querySelector("#button").style.left')).toBe('0px');
     });
   });
+
+  describe('Page.check', function() {
```

Comment:
We should follow up with checking radio buttons (IIRC, WebDriver does it).

Reply:
Added radio into the mix